pyproject.toml

```diff
@@ -106,6 +106,7 @@ module = [
     'arelle.plugin.validate.ESEF.*',
     'arelle.Cntlr',
     'arelle.Locale',
+    'arelle.ModelValue',
```

Comment:
nit: sort is off

Reply:
@austinmatherne-wk
I am waiting on #342 to merge to resolve any other conflicts here in one go.

Reply:
@austinmatherne-wk 
aside note, seems that the conformance suites are running on ubuntu only, is that the intention?

Reply:
Yes. We considered running on multiple platforms, but decided against it due to the increased execution time.